aws-rds-dbcluster/src/main/java/software/amazon/rds/dbcluster/Translator.java

```diff
@@ -203,7 +205,18 @@ static ModifyDbClusterRequest modifyDbClusterRequest(
                 builder.allowMajorVersionUpgrade(true);
             }
         }
-
+        //only include VPC SG ids if they are changed from previous.
+        Set<String> desiredVpcSgIds = streamOfOrEmpty(desiredModel.getVpcSecurityGroupIds()).collect(Collectors.toSet());
+        if (!desiredVpcSgIds.isEmpty()) {
+            if (previousModel != null) {
+                Set<String> previousVpcSgIds = streamOfOrEmpty(previousModel.getVpcSecurityGroupIds()).collect(Collectors.toSet());
+                if (desiredVpcSgIds.size() != previousVpcSgIds.size() || !desiredVpcSgIds.containsAll(previousVpcSgIds)) {
+                    builder.vpcSecurityGroupIds(desiredModel.getVpcSecurityGroupIds());
+                }
```

Comment:
Let's simplify this bit with a helper. It seems we have 2 conditional branches leading to setting vpc sec groups to `desiredModel.getVpcSecurityGroupIds()`. Could be there a mistake? If no, shall we unify them?